src/server.rs

```diff
@@ -34,6 +34,11 @@ pub use self::{
     node_types::{MethodNode, Node, ObjectNode, VariableNode},
 };
 
+#[cfg(feature = "tokio")]
+use tokio_util::sync::CancellationToken;
+#[cfg(feature = "tokio")]
+use tokio_util::task::TaskTracker;
```

Comment:
I suggest to avoid imports behind feature flags.

Either move the code into a separate module (hidden behind the feature flag, would be overkill in this case), or import the symbols in the local context (= function), or use explicit qualifications.